packages/playwright-crawler/src/internals/playwright-crawler.ts

```diff
@@ -201,6 +201,10 @@ export class PlaywrightCrawler extends BrowserCrawler<{ browserPlugins: [Playwri
                 + 'Use PlaywrightCrawlerOptions.proxyConfiguration');
         }
 
+        if (browserPoolOptions.browserPlugins) {
```

Comment:
this seems breaking, idk how valid the option was, or if its supposed to be purely internal?

cc @mnmkng

Reply:
This was never working, it's to prevent mistakes.

Reply:
ok i see, then we might as well add `@deprecated` jsdoc tag on that interface property 

i'd still like to see a confirmation for that, maybe there is some use case we dont know about

Reply:
`browserPlugins` is working when it's not overriden by `launcher`, which for crawlers it is always overriden. Hence the error to use the other option.

Reply:
ok, now i get it. lets add a comment saying exactly this

Reply:
commented.